### PR TITLE
[Release] linux_sql

### PR DESCRIPTION
**Pull request by adams-liu**
Hi, 

Can you please review my code!

Thanks!

### CODE REVIEW

File: linux_sql/scripts/psql_docker.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#error to check if jrvs-psql container exists
+if [ $(docker container ls -a -f name=jrvs-psql | wc -l) -eq 1 ];then
+  echo "jrvs-psql does not exist"
+  exit 1
+fi
+
+#start jrvs-psql container
+if [ "$1" == "start" ];then
+  docker container start jrvs-psql
+  exit $?
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:36]: what happens if the container was not created

File: linux_sql/scripts/psql_docker.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+fi
+
+#stop jrvs-psql container
+if [ "$1" == "stop" ];then
+  docker container stop jrvs-psql
+  exit $?
+fi
+
+#error to check if command exists
+if [ "$1" != "create" || "$1" != "start" || "$1" != "stop" ];then
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh]: if the statement is not necessary

File: linux_sql/scripts/psql_docker.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  echo "invalid command"
+  exit 1
+fi
+
+exit 0
```
Comment (jarviscanada) [inline linux_sql/scripts/psql_docker.sh:45]: not necessary

**Review by jarviscanada — APPROVED** [review]
fix all and merge

File: linux_sql/scripts/psql_docker.sh (hunk 1 above)
Comment (adams-liu) [reply linux_sql/scripts/psql_docker.sh:36]: doesn't line 27 - 31 work for this then? I'm unsure what I am missing?

#error to check if jrvs-psql container exists
if [ $(docker container ls -a -f name=jrvs-psql | wc -l) -eq 1 ];then
  echo "jrvs-psql does not exist"
  exit 1
fi

**Pull request merged**